src/mongo/db/pipeline/document_source.h

```diff
@@ -1485,7 +1485,9 @@ class DocumentSourceLookUp final : public DocumentSource,
                                                    Pipeline::SourceContainer* container) final;
     GetDepsReturn getDependencies(DepsTracker* deps) const final;
     void dispose() final;
```

Comment:
Now that getOutputSorts() has an inline definition, please add a newline after this line.